models/staging/stg_fivetran_platform__connection.sql

```diff
@@ -0,0 +1,117 @@
+/* 
+The CONNECTOR source is deprecated in favor of CONNECTION.
+This model merges data from both source tables, handling any combination of the tables being present (either one or both).
+The configuration variables `fivetran_platform_using_connection` and `fivetran_platform_using_connector` is ideally set 
+by Quickstart, but a core user can also customize. 
+It combines records from both sources into a single output and deduplicates across the two sources.
+This logic will be removed when the CONNECTOR source is removed.
```

Comment:
Normally we choose a source when there is a rename, but I chose to do this mainly because it's our Fivetran Platform. It didn't make sense for users to have to combine the sources on their own or lose data for something we do have control over (unlike other connector source names).